RadarSDK/RadarLocationManager.m

```diff
@@ -578,11 +629,35 @@ - (void)sendLocation:(CLLocation *)location stopped:(BOOL)stopped source:(RadarL
 
     self.sending = YES;
 
+    NSArray<NSString *> *nearbyBeacons;
+    RadarTrackingOptions *options = [RadarSettings trackingOptions];
+    if (options.beacons) {
+        nearbyBeacons = [self.nearbyBeaconIdentifers allObjects];
+
+        [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug
+                                           message:[NSString stringWithFormat:@"Sending nearby beacons | nearbyBeacons = %@", [nearbyBeacons componentsJoinedByString:@","]]];
+
+        if (source != RadarLocationSourceBeaconEnter && source != RadarLocationSourceBeaconExit && source != RadarLocationSourceMockLocation &&
+            source != RadarLocationSourceManualLocation) {
+            [[RadarAPIClient sharedInstance] searchBeaconsNear:location
```

Comment:
probably worth a comment here explaining why we search for beacons at this point.

Reply:
May be worth formalizing best practices and a style guide for comments in our open-source SDKs, then having me come through through and comment everything in this class thoroughly and consistently in a separate PR (alternatively, documenting it even more thoroughly internally). Let's discuss offline this week